pkg/buildbot_pkg.py

```diff
@@ -205,9 +205,13 @@ def run(self):
 
             yarn_program = None
             for program in ["yarnpkg", "yarn"]:
-                yarn_version = check_output([program, "--version"], shell=shell)
-                if yarn_version != "":
-                    yarn_program = program
+                try:
```

Comment:
checkoutput raise in case of failure, which would be the case for at least one of those two.

Reply:
Right, I have both installed...